package gzmaze.msgs;
import "pose.proto";

message RobotState
{
  required gazebo.msgs.Pose    pose    = 1;
  required float               left_wheel_velocity  = 2;
  required float               right_wheel_velocity = 3;
  required float               left_wheel_angle  = 4;
  required float               right_wheel_angle = 5;
  required float               left_analog          = 6;
  required float               right_analog         = 7;
  required bool                left_binary          = 8;
  required bool                right_binary         = 9;
  required bool                front_binary         = 10;

}
